kubejs/assets/kubejs/lang/zh_cn.json

```
@@ -47,24 +47,24 @@
     "item.kubejs.incomplete_linen_thread_from_scrap": "Incomplete Linen Thread from Scrap",
     "item.kubejs.incomplete_linen_thread_from_flax": "Incomplete Linen Thread from Flax",
 
-    "item.kubejs.hot_queens_slime_ingot": "Hot Queen's Slime Ingot",
-    "item.kubejs.hot_hepatizon_ingot": "Hot Hepatizon Ingot",
-    "item.kubejs.hot_manyullyn_ingot": "Hot Manyullyn Ingot",
+    "item.kubejs.hot_queens_slime_ingot": "热皇后史莱姆锭",
+    "item.kubejs.hot_hepatizon_ingot": "热黑色科林斯青铜锭",
+    "item.kubejs.hot_manyullyn_ingot": "热玛玉灵锭",
 
-    "item.kubejs.ender_sushi": "Ender Sushi",
-    "item.kubejs.ender_sushi.tooltip": "Eat me to unlock access to The End",
+    "item.kubejs.ender_sushi": "末影寿司",
+    "item.kubejs.ender_sushi.tooltip": "食用后获得进入末地的权限",
```

https://minecraft.fandom.com/zh/wiki/%E6%9C%AB%E5%9C%B0?variant=zh
```suggestion
    "item.kubejs.ender_sushi.tooltip": "食用后获得前往末地的权限",
```